template/node10-express/function/handler.js

```diff
@@ -1,12 +1,64 @@
 "use strict"
 
-module.exports = (event, context) => {
-    let err;
+// callback version
+const delayCallback = cb => {
```

Comment:
This is now part of the main template, was that intended or was it a mistake?

Reply:
I see handler.js as an example on how to build the code for the serverless function. As the main motivation for this refactoring is to handle asynchronicity, I use that function to simulate some load, and show how to take advantage of this new feature. But it's just a proposal, you can keep it or use the original example.